reactquiz/src/components/quiz/Results.js

```diff
@@ -1,34 +1,25 @@
-import React,{Component} from 'react';
-
-
-class Results extends Component{
-
+import React, { Component } from 'react';
 
```

Comment:
Adding a comment about the purpose of this component or how it fits into the overall application would be helpful for context.